### ISSUE TITLE
Offline support

### ISSUE THREAD

**Issue by Jonas12354** [meteor-utilities/avatar#50]
Hi, 
is there any possibility to have an offline support for the avatar?
Obviously, the app cannot access the url for the profile picture when not online. Did someone try to download the avatar to the database or something like that? Is it possible to cache avatar pictures from an external url? 
Thanks for your help.

**Comment by timothyarmes** [comment]
Hi,

If you're not online then you won't even be able to access the Meteor Server...

**Comment by Jonas12354** [comment]
That's correct. But I grounded the Meteor.users database with ground:db and gwendall:ground-user package. So all the data is available offline, even without a connection to the server.

**Comment by timothyarmes** [comment]
Cool. PR’s welcomed then ;)
ᐧ

**Comment by blahah** [comment]
Just adding that Meteor apps can be deployed as electron desktop apps, or run on the localhost or over a local network :)
